### PR TITLE
EventFilter/SiStripRawToDigi: clean up clang warnings:

### PR DESCRIPTION
**Pull request by gartung**
src/EventFilter/SiStripRawToDigi/interface/SiStripFEDBuffer.h:39:20: warning: 'sistrip::FEDBuffer::channelGood' hides overloaded virtual function [-Woverloaded-virtual]
       virtual bool channelGood(const uint8_t internalFEDannelNum, const bool doAPVeCheck=true) const;
                   ^
src/EventFilter/SiStripRawToDigi/interface/SiStripFEDBuffer.h:44:20: warning: 'sistrip::FEDBuffer::doChecks' hides overloaded virtual function [-Woverloaded-virtual]
       virtual bool doChecks(bool doCRC=true) const;
                   ^

### CODE REVIEW

**Comment by gartung** [comment]
@Dr15Jones

File: EventFilter/SiStripRawToDigi/plugins/SiStripRawToDigiUnpacker.cc [hunk 1]
```diff
@@ -181,7 +181,7 @@ namespace sistrip {
       try {
         buffer.reset(new sistrip::FEDBuffer(input.data(),input.size()));
         buffer->setLegacyMode(legacy_);
-        if (!buffer->doChecks()) {
+        if (!buffer->doChecks(bool(1))) {
```
Comment (Dr15Jones) [inline EventFilter/SiStripRawToDigi/plugins/SiStripRawToDigiUnpacker.cc]: How about `!buffer->doChecks(true)`

**Comment by gartung** [comment]
@Dr15Jones  code-checks caught that

**Comment by Dr15Jones** [comment]
please test

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by jfernan2** [comment]
+1

**Comment by lpernie** [comment]
+1

**Comment by fabiocos** [comment]
+1

technical code fixes, no evident effect in test workflows

**Pull request merged**